### ISSUE TITLE
some fcp1000 datasets didn't crawl correctly

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#655]
```
Cleveland CCF                    [annex]  incoming  - -  X  0 Bytes/0 Bytes
Durham_Madden                    [annex]  incoming  - -  X  0 Bytes/0 Bytes
NewYork_Test-Retest_Reliability  [annex]  incoming  - -  X  0 Bytes/0 Bytes
```
you can find them under `/mnt/datasets/datalad/crawl/indi/fcon1000` or just recrawl in your version... didn't look into detail yet

**Comment by yarikoptic** [comment]
ok -- tried in Cleveland
```
(git)smaug:/mnt/datasets/datalad/crawl/indi/fcon1000/Cleveland CCF[master]git
$> datalad crawl
2016-07-20 00:26:23,601 [INFO   ] Loading pipeline specification from ./.datalad/crawl/crawl.cfg (crawl.py:103)
2016-07-20 00:26:23,933 [INFO   ] Creating a pipeline for the FCP Table dataset Cleveland CCF (fcptable.py:93)
2016-07-20 00:26:24,052 [INFO   ] Running pipeline [<function switch_branch at 0x7f7d2c1b2ed8>, [<datalad.crawler.nodes.crawl_url.crawl_url object at 0x7f7d2c1bfbd0>, [assign(assignments={'dataset': 'Cleveland CCF'}, interpolate=False), sub(ok_missing=False, subs=<<{'response': {'<div cl...>>), <datalad.crawler.pipelines.fcptable.find_dataset object at 0x7f7d2c1bfc50>, [<function extract_readme at 0x7f7d2c1aded8>, <datalad.crawler.nodes.annex.Annexificator object at 0x7f7d32b9db50>], <datalad.crawler.nodes.matches.a_href_match object at 0x7f7d2c1bfc90>, <datalad.crawler.nodes.annex.Annexificator object at 0x7f7d32b9db50>]], <datalad.crawler.nodes.annex._remove_obsolete object at 0x7f7d2c1bfcd0>, [<function switch_branch at 0x7f7d2c1b2f50>, <function merge_branch at 0x7f7d2bd02140>, [{'loop': True}, find_files(dirs=False, fail_if_none=True, regex='\\.(zip|tgz|tar(\\..+)?)$', topdir='.'), <function _add_archive_content at 0x7f7d2bd021b8>], <function switch_branch at 0x7f7d2bd02230>, <function merge_branch at 0x7f7d2bd022a8>, <function _finalize at 0x7f7d2bd02320>], <function switch_branch at 0x7f7d2bd02398>, <function _finalize at 0x7f7d2bd02410>] (crawl.py:106)
2016-07-20 00:26:24,126 [INFO   ] Found branch non-dirty - nothing is committed (annex.py:1110)
2016-07-20 00:26:24,127 [INFO   ] Checking out a new detached branch incoming (annex.py:668)
2016-07-20 00:26:24,162 [INFO   ] Fetching 'http://fcon_1000.projects.nitrc.org/fcpClassic/FcpTable.html' (base.py:461)
2016-07-20 00:26:24,176 [INFO   ] Generated README.txt (fcptable.py:52)
2016-07-20 00:26:24,295 [ERROR  ] Did not match required 1 matches (got 0) using <datalad.crawler.nodes.matches.a_href_match object at 0x7f7d2c1bfc90> [matches.py:__call__:100] (ValueError) (main.py:246)
```

**Comment by glalteva** [comment]
Cleveland CCF and NewYork_Test-Retest_Reliability don't have download links to match- their links redirect to different pages (which do contain tarballs, although the New York one looks a bit tricky). Durham_Madden doesn't have a link at all.

**Comment by glalteva** [comment]
Closed by: #670
